products/sqlite.md

```diff
@@ -0,0 +1,349 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+alternate_urls:
+-   /sqlite3
```

Comment:
Should be removed : [sqlite3](https://docs.python.org/3/library/sqlite3.html#module-sqlite3) is a python library.

Reply:
I was wrong, see https://en.wikipedia.org/wiki/SQLite.